public/buttons.html

```diff
@@ -51,7 +51,14 @@ <h1>PayPal Buttons Example</h1>
                 }),
               })
                 .then((response) => response.json())
-                .then((order) => order.id);
+                .then((orderResponse) => {
```

Comment:
I wonder if we should add a `catch()` here and enhance the error message coming back from the server to make sure its clear that it originated from the `createOrder()` callback. Let me know your thoughts.

Reply:
I ended up adding the `catch()` to prefix the error message.